src/context.rs

```diff
@@ -96,19 +83,22 @@ impl<'a> Context<'a> {
 
     /// Create a new module
     ///
-    /// Will return `None` if the module is disabled by configuration, by setting
-    /// the `disabled` key to `true` in the configuration for that module.
+    /// Can always return a new module as if a module was disabled it won't get
+    /// to this point.
     pub fn new_module(&self, name: &str) -> Option<Module> {
```

Comment:
I suppose this could return `Module` rather than `Option<Module>` now?

Reply:
I think I avoided this initially because I would have to touch every module, but maybe we should do that anyways.

Reply:
Modules still do return `None` when there's nothing relevant to show. For example, the Node.js module when there are no relevant files.

Reply:
This specific function can return just `Module`. Each `Module::module` function is still returning Option<Module>`, so we don't lose the functionality. We save a character in each module by removing the `?` from the end of `let module = context.new_module("");` call.